scripts/core/data/users/UserActions.ts

```diff
@@ -0,0 +1,21 @@
+import {IUser} from 'superdesk-api';
+import SuperdeskDispatcher from '..';
+import UserActionTypes from './UserActionTypes';
+
+const Actions = {
+    updateUser(data: IUser) {
+        SuperdeskDispatcher.dispatch({
+            type: UserActionTypes.UPDATE_USERS,
+            payload: data,
+        });
+    },
+
+    initUsers(data: Array<IUser>) {
+        SuperdeskDispatcher.dispatch({
+            type: UserActionTypes.INIT_USERS,
+            payload: data,
+        });
+    },
+};
+
+export default Actions;
```

Comment:
I'd use named exports everywhere. It enforces consistency of naming across the app and still allows for overwriting to handle cases when the name is taken by another variable.